docs/BasicEventEntryTypes.md

```diff
@@ -0,0 +1,136 @@
+
+```json
+{
+  *"type": "ambrosus.asset.info",
+  "assetType": "...",
+  *"name": "...",
+  "localisedName": {
+    "...": "..."
+	},
+  "description": "...",
+  "localisedDescription": {
+    "...": "..."
+  },
+  "tags":["...","..."],
+  "images": {
+    "default": {
+      *"url": "http://...",
+      "size": {
+        *"width": 300,
+        *"height": 600
+      }
+    },
+		"...": {
+      *"url": "http://...",
+      "size": {
+        *"width": 300,
+        *"height": 600
+      }
+    }
+  }
+}
+```
+
+`assetType` is the type of the asset (such as a pallet, a box, an invoice, a batch, a unique item, etc.). Ambrosus provides the following types: `ambrosus.assetTypes.item` (for individual items with a unique serial number), `ambrosus.assetTypes.batch` (for a batch/lot of a certain), `ambrosus.assetTypes.sku` ("Stock Keeping Unit" used for a class of products that share the same attributes such as color, size, size, brand). Apps can create custom asset types as required. 
+
+`name` is the default human readable name of the product (in English). Additionally localised versions of name can be provided using the `localisedName` dictionary, where the key is the ISO 639-1 language code, and the value holds the translation. 
+
+`description` is the default human readable description of the asset (in English). Additionally localised versions of it can be provided using the `localisedDescription` dictionary, where the key is the ISO 639-1 language code, and the value holds the translation. (same as for name)
+
+`tags` is an array of strings (`"tags":["test", "local", "UK"]`).
+
+`images` is a dictionary that maps identifiers (this can be used to distinguish sizes, orientations, etc.) to objects describing images. Each such object must have a `url` field. Adding a optional `size` object with `width` and `height` is recommended. Additional fields are permitted but will not be checked/validated. For example:
+
+```json
+{
+  "...",
+  "images": {
+    "default": {
+      *"url": "https://5.imimg.com/data5/WQ/JP/MY-10901605/coffee-packaging-bags-500x500.jpg",
+      "name": "Main product image",
+      "size": {
+        *"width": 300,
+        *"height": 600
+      }
+    },
+    "second": {
+      "url": "https://5.imimg.com/data5/WQ/JP/MY-10901605/coffee-packaging-bags-500x500.jpg",
+      "name": "Alternative product image",
+			"size": {
+        *"width": 300,
+        *"height": 600
+      }
+    },
+    "mobile": {...}
+  }
+}
+```
+
+Additional fields to the `ambrosus.asset.info` entry type are allowed, but will not be checked/validated. 
+
+### Asset/Event Identifier
+
+The different identifiers for an asset. Can represent a unique item (e.g. unique serial ID) or a identifier for a type of item (e.g. barcode).
+
+```json
+{
+  *"type": "ambrosus.asset.identifiers",
+  *"identifiers": {
+    "ean13": ["...."],
```

Comment:
Maybe write `"ean13": ["....", "...."]` instead of `"ean13": ["...."]` to underline the fact, that this arrays can contain more than one item?